Updates to risk block and codelists

duncandewhurst opened the pull request:
Make riskCategory an open codelist and update codes and definitions in
line with consultation feedback. Remove likelihood and fiscalImpact
properties. Remove shared code from riskAllocation codelist and replace
with 'undefined'

duncandewhurst committed:
Updates to risk block and codelists

Make riskCategory an open codelist and update codes and definitions in
line with consultation feedback. Remove likelihood and fiscalImpact
properties. Remove shared code from riskAllocation codelist and replace
with 'undefined'

timgdavies reviewed (changes requested):
To fit with 1.1 we need to:

(1) Remove the merge strategies;

(2) Include 

``` "codelist":"codelist.csv", 
      "openCodelist":false 
```
properties against each field with codes, and remove the enums.

duncandewhurst committed:
Align with OCDS V1.1

Remove mergeStrategy for all properties.
Remove enum for allocation codelist and add "openCodelist": "false"

duncandewhurst committed:
riskCategory codelist updates

Update codelist descriptions and remove 'other' code since riskCategory
is an open codelist.

duncandewhurst commented:
@timgdavies I've made those updates in [452b123](https://github.com/open-contracting/ocds-riskAllocation-extension/pull/2/commits/452b123742cd72ce2f4ef7d1218fb45f7e7c4b3f) and also update the riskCategory codelist in line with Abdou's latest email.

Ok to merge now?

timgdavies commented on release-schema.json:
```diff
@@ -11,40 +11,23 @@
-            null
-          ],
-          "mergeStrategy": "ocdsVersion"
+          ]
```
add

```json
"codelist":"riskCategory.csv",
"openCodelist":false
```

timgdavies commented on release-schema.json:
```diff
@@ -53,61 +36,23 @@
-            null
-          ],
-          "mergeStrategy": "ocdsVersion"
+		"openCodelist": "false"
```
Add

```json
"codelist":"riskAllocation.csv",
"openCodelist":false
```

(note no quotes around false)

timgdavies reviewed (changes requested):
Small tweaks still needed to schema I'm afraid. Check the 1.1 scheme for the syntax for codelist markup etc. http://standard.open-contracting.org/1.1-dev/en/release-schema.json

duncandewhurst committed:
Update codelists in schema

Add references to .csvs and set openCodelist flags for riskCategory and
riskAllocation codelists.

duncandewhurst commented:
@timgdavies thanks - updated inline with your comments, except for riskCategory which is an open codelist so have set openCodelist = true
